superset-frontend/src/explore/actions/exploreActions.ts

```diff
@@ -86,7 +86,7 @@ export function fetchFaveStar(sliceId: string) {
 
 export const SAVE_FAVE_STAR = 'SAVE_FAVE_STAR';
 export function saveFaveStar(sliceId: string, isStarred: boolean) {
-  return function (dispatch: Dispatch<ReturnType<typeof addDangerToast>>) {
+  return function (dispatch: Dispatch) {
```

Comment:
why do you remove return type?

Reply:
This was a while ago so I can't _quite_ remember, but I think the redux upgrade made the extra typing unnecessary.

Reply:
Okay, I ran into this again while upgrading redux in a separate branch, and the reason is that the `Dispatch` here is being used with more than just `addDangerToast` - it's also being used to dispatch the `toggleFaveStar` action. It appears that redux have upgraded their types, and I don't think it really makes sense to type Dispatch arguments like this anymore.

Reply:
At least, not without more robust typings on our Actions and on the Store. You can see an example of well-typed reducers and actions in the `nativeFilters` actions and reducers added in this PR. Arranging types in a way that doesn't make development difficult is kind of tricky.